test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,79 @@
+./test_scripts/API/SetAudioStreamingIndicator/001_ATF_N_TC_SetAudioStreamingIndicator_PLAY_PAUSE_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PLAY_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_PLAY_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PLAY
+./test_scripts/API/SetAudioStreamingIndicator/003_ATF_N_TC_SetAudioStreamingIndicator_PAUSE_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/004_ATF_N_TC_SetAudioStreamingIndicator_PLAY_PAUSE_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PLAY_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/005_ATF_N_TC_SetAudioStreamingIndicator_PLAY_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PLAY
+./test_scripts/API/SetAudioStreamingIndicator/006_ATF_N_TC_SetAudioStreamingIndicator_PAUSE_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/007_ATF_N_TC_SetAudioStreamingIndicator_non-media_app_REJECTED.lua Check_that_SDL_sends_REJECTED_to_non-media_app_for_RPC_SetAudioStreamingIndicator
+./test_scripts/API/SetAudioStreamingIndicator/008_ATF_N_TC_SetAudioStreamingIndicator_navi_app_REJECTED.lua Check_that_SDL_sends_REJECTED_to_navi_app_for_RPC_SetAudioStreamingIndicator
+./test_scripts/API/SetAudioStreamingIndicator/009_ATF_N_TC_SetAudioStreamingIndicator_voice_com_app_REJECTED.lua Check_that_SDL_sends_REJECTED_to_voice_com_app_for_RPC_SetAudioStreamingIndicator
+./test_scripts/API/SetAudioStreamingIndicator/010_ATF_N_TC_SetAudioStreamingIndicator_non-media_app_REJECTED_fake_param.lua Check_that_SDL_sends_REJECTED_to_non-media_app_for_RPC_SetAudioStreamingIndicator_fake_param_from_other_RPC
+./test_scripts/API/SetAudioStreamingIndicator/011_ATF_N_TC_SetAudioStreamingIndicator_navi_app_REJECTED_fake_param.lua Check_that_SDL_sends_REJECTED_to_navi_app_for_RPC_SetAudioStreamingIndicator_fake_param_from_other_RPC
+./test_scripts/API/SetAudioStreamingIndicator/012_ATF_N_TC_SetAudioStreamingIndicator_voice_com_app_REJECTED_fake_param.lua Check_that_SDL_sends_REJECTED_to_voice_com_app_for_RPC_SetAudioStreamingIndicator_fake_param_from_other_RPC
```

Comment:
@OVikhrov, agree that can be merged in one script, but prefer to be separated because are checking different kind of applications(non-media, navi, voice-com) and the implementation for these types can be different.
So I prefer to leave scripts in this way.